### PR TITLE
test(swap): add tests for fixes

### PR DESCRIPTION
**Pull request by acdibble**

### CODE REVIEW

File: packages/swap/src/routes/__tests__/v2/quote.test.ts [hunk 1]
```diff
@@ -53,6 +53,7 @@ jest.mock('../../../pricing/checkPriceWarning', () => ({
 describe(getDcaQuoteParams, () => {
   beforeEach(() => {
     jest.clearAllMocks();
+    jest.restoreAllMocks();
```
Comment (acdibble) [inline packages/swap/src/routes/__tests__/v2/quote.test.ts:56]: this didn't even do anything. am i joke to you?

**Review by jerryafr — APPROVED** [review]
🙏

File: packages/swap/src/routes/__tests__/v2/quote.test.ts (hunk 1 above)
Comment (niklasnatter) [reply packages/swap/src/routes/__tests__/v2/quote.test.ts:56]: i never understood when to use these methods

File: packages/swap/src/routes/__tests__/v2/quote.test.ts [hunk 2]
```diff
@@ -1147,22 +1139,13 @@ describe('server', () => {
...
-      const sendSpy = jest
-        .spyOn(WsClient.prototype, 'sendRequest')
-        .mockResolvedValueOnce({
-          ingress_fee: buildFee('Eth', 25000).bigint,
-          egress_fee: buildFee('Usdc', 0).bigint,
-          network_fee: buildFee('Usdc', 100100).bigint,
-          intermediary: null,
-          output: BigInt(100e6),
-        })
-        .mockResolvedValueOnce({
```
[the remark this answers is not in the document]
Comment (niklasnatter) [reply packages/swap/src/routes/__tests__/v2/quote.test.ts:1159]: i had to do it 4 times in the test i wrote yesterday - maybe we need to adjust that too?

**Pull request merged**